<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;温度检测系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>MULTIBEANS技术支持：QQ380499998；毕业设计代做；论文发表；</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>信息提示：</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
font: 8pt &quot;黑体&quot;;
color: rgb(85, 255, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>控制显示</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLCDNumber" name="lcdNumber">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>80</height>
               </size>
              </property>
              <property name="digitCount">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>温度补偿</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QComboBox" name="DeviceList">
                  <item>
                   <property name="text">
                    <string>自动连接</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string/>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_startDevice">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>开启设备</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_stopDevice">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>关闭设备</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_light">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>指示灯</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_diatemp">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDial" name="dial">
                  <property name="minimum">
                   <number>100</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>绘图</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCustomPlot" name="plot" native="true">
          <property name="minimumSize">
           <size>
            <width>950</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>820</width>
            <height>250</height>
           </size>
          </property>
          <widget class="QLabel" name="label_currentTemp">
           <property name="geometry">
            <rect>
             <x>810</x>
             <y>20</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>0.0℃</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
